caller/train_crf_rnn_loop.sh.dist

```diff
@@ -0,0 +1,18 @@
+export THEANO_FLAGS=mode=FAST_RUN,floatX=float32,device=cuda,force_device=True,exception_verbosity=high
+
+if [ "$#" -lt 1 ]
+then
+    echo "You must provide the amount of training loops as a script argument! Aborting." 1>&2
+    exit 1
+fi
+
+loop_max=$1
+for loop_index in $(seq 1 "$loop_max")
+do
+    training_id="$(date +%Y-%m-%d_%H:%M:%S)__${loop_index}"
```

Comment:
Why not declare the entire monitoring- and eval-files here as variables? That would make the calling string in line 16 much shorter...